server/components/populate_sport_test.js

```diff
@@ -3,37 +3,40 @@
 **/
 
 var Converter = require('csvtojson').core.Converter;
-var db = require('../server/api/models');
+var db = require('../api/models');
 var Sport = db.Sport;
 var Player = db.Player;
 var fs = require('fs')
+var BPromise = require("sequelize/node_modules/bluebird");
 
 var Populate = (function(){
   var _parseCsv = function(csv, sport) {
     var fileStream = fs.createReadStream(csv);
     var csvConverter = new Converter({constructResult:true});
-    csvConverter.on("end_parsed",function(jsonObj){
-      populateDatabase(jsonObj, sport);
+    return BPromise.try(function() {
```

Comment:
I'm trying to fix the promise chain in this spec, which I think always gets broken when the CSV parser is run. I want to be able to actually have the promise chain incorporate the csv parser and wait until it finishes running so the spec works properly (in the sense that the chain isn't broken up at all so the first spec doesn't run until this is finished.)